externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/shim/SparkCatalogShim.scala

```diff
@@ -123,7 +123,7 @@ trait SparkCatalogShim extends Logging {
       methodName: String,
       args: (Class[_], AnyRef)*): Any = {
     val (types, values) = args.unzip
-    val method = obj.getClass.getDeclaredMethod(methodName, types: _*)
+    val method = obj.getClass.getMethod(methodName, types: _*)
```

Comment:
`getDeclaredMethods` includes all methods declared by the class itself, whereas `getMethods` returns only public methods, but also those inherited from a base class.